src/parser/lval.js

```diff
@@ -123,14 +123,11 @@ pp.parseBindingIdentifier = function () {
 };
 
 // Parses lvalue (assignable) atom.
-
 pp.parseBindingAtom = function () {
   switch (this.state.type) {
     case tt._yield:
-      if (this.state.strict || this.state.inGenerator) this.unexpected();
```

Comment:
Yes I was just worried because I couldn't find a check for isGenerator but I see now that when it is a generator it will check for keywords inside parseIdentifier()